src/Explore/Detailed.js

```diff
@@ -40,29 +47,47 @@ function Table({ table }) {
     </Flex>
   )
 }
+
 function Title({ expand, onClick, text }) {
   return (
     <Text onClick={onClick} fontWeight="bold">
       {`${text} ${expand ? '\u2227' : '\u2228'}`}
     </Text>
   )
 }
-function Details(props) {
-  const { open, title } = props
-  const [expanded, toggle] = useToggle(open)
+
+function Detailed(props) {
+  const { pid, summary, releaseDate } = props
+  const [expanded, toggle] = useToggle(false)
+
   return (
-    <Box
-      sx={{
-        cursor: 'pointer',
-      }}
-      onClick={() => toggle()}
-    >
-      <Title expand={expanded} text={title} />
-      <Suspense fallback={<Spinner />}>
-        {expanded && <Table table={props} />}
-      </Suspense>
-    </Box>
+    <>
+      <Box as="article">{summary}</Box>
+      <br />
+      <Text sx={{ fontWeight: 'bold' }}>
+        Released: {parseDate(releaseDate)}
+      </Text>
+      <Box sx={{ cursor: 'pointer' }} onClick={() => toggle()}>
+        <Title expand={expanded} text="Members" />
+        <Suspense fallback={<Spinner />}>
+          {expanded && (
+            <Table
+              table={{
+                columns: [
+                  ['Person', 'fullName'],
+                  ['Affiliation', 'name'],
+                ],
+                url: `/documents/${encodeURIComponent(pid)}`,
+                mergeFn: getMembers,
+                rowId: 'id',
+                config: { include: ['person', 'affiliation'] },
+              }}
+            />
+          )}
+        </Suspense>
+      </Box>
+    </>
   )
 }
 
-export default Details
+export default Detailed
```

Comment:
I've moved the `Detailed` component that was in `DocumentItem.js` here and merged it into the `Details` component. I've then kept the name `Detailed`.